purr/styles/verdigris.json

```diff
@@ -0,0 +1,149 @@
+{
+	"$schema": "https://schemas.wp.org/wp/6.5/theme.json",
+	"version": 2,
+	"title": "Verdigris",
+	"settings": {
+		"color": {
+			"palette": [
+				{
+					"color": "#000000",
+					"name": "Base",
+					"slug": "base"
+				},
+				{
+					"color": "#32a852",
```

Comment:
I think the colors in both variations have too low contrast, have a look at the [accessibility guidelines for themes](https://developer.wordpress.org/themes/functionality/accessibility/#color-and-color-contrast)